clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/UBamFileStatus.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.clio.transfer.model
```

Comment:
I've named it DocumentStatus (already had that refactor in place) - we can discuss if FileStatus is a better choice.